app/forms/journeys/additional_payments_for_teaching/claim_submission_form.rb

```diff
@@ -1,36 +1,15 @@
 module Journeys
   module AdditionalPaymentsForTeaching
     class ClaimSubmissionForm < ::ClaimSubmissionBaseForm
-      ELIGIBILITY_ATTRIBUTES = [
-        [:nqt_in_academic_year_after_itt, :boolean],
-        [:employed_as_supply_teacher, :boolean],
-        [:qualification, :string],
-        [:has_entire_term_contract, :boolean],
-        [:employed_directly, :boolean],
-        [:subject_to_disciplinary_action, :boolean],
-        [:subject_to_formal_performance_action, :boolean],
-        [:eligible_itt_subject, :string],
-        [:eligible_degree_subject, :string],
-        [:teaching_subject_now, :boolean],
-        [:itt_academic_year, AcademicYear::Type.new],
-        [:current_school_id, :string], # uuid
-        [:induction_completed, :boolean],
-        [:school_somewhere_else, :boolean]
-      ]
-
-      ELIGIBILITY_ATTRIBUTES.each do |name, type|
-        attribute name, type
-      end
-
       private
 
-      def eligibility_attribute_names
-        ELIGIBILITY_ATTRIBUTES.map(&:first)
+      def journey
+        Journeys::AdditionalPaymentsForTeaching
       end
```

Comment:
I'm not sure if this is necessary as this class lives in the relevant journey namespace. If we still need the class to tell us this (I'm not sure where it's used) we could potentially implement it in `ClaimSubmissionBaseForm` using `Module.nesting.first`.

Reply:
It's used to find the policies we need to build eligibilities. I've now moved it into the base form as we can work it out from there 👍